src/Config_Command.php

```diff
@@ -856,4 +864,20 @@ private function parse_separator( $separator ) {
 
 		return $separator;
 	}
+
+	/**
+	 * Writes a provided variable's key and value to stdout, in dotenv format.
+	 *
+	 * @param array $value
+	 */
+	private function print_dotenv ( array $value )
+	{
+		if ( ! isset( $value['name'] ) || ! isset( $value['type'] ) || 'constant' !== $value['type'] ) {
```

Comment:
@schlessera Do you agree that only constant should be supported here?

Reply:
Yes, I agree.